tap_helpshift/sync.py

```diff
@@ -13,7 +13,7 @@ async def sync_stream(state, instance, counter, *args, start_date=None):
     LOGGER.info("%s: Starting sync", stream_name)
 
     # If we have a bookmark, use it; otherwise use start_date & update bookmark with it
-    if (instance.replication_method == 'INCREMENTAL' and
+    if (instance.use_bookmarks and instance.replication_method == 'INCREMENTAL' and
```

Comment:
We should continue writing bookmarks even if the current sync didn't read from the bookmarks, because that will allow a different sync that does read bookmarks to skip duplicate work.

Reply:
Wouldn't runs past the first one not be synced properly again? Because we're looking at the bookmarks or the start date, and after the first run, it won't consider the start date anymore. Correct me if I'm wrong, but I think we want to look at the bookmarks while it's running and look at the start date while the tap is starting. If we're planning to have this full sync every run, then it's like we're clearing the state whenever the tap starts syncing?